internal/pkg/exec/docker.go

```diff
@@ -52,12 +55,17 @@ const (
 	credStoreECRLogin = "ecr-login" // set on `credStore` attribute in docker configuration file
 )
 
-// Architectures that receive special handling.
+// Operating systems and architectures.
 const (
-	ArmArch   = "arm"
-	Arm64Arch = "arm64"
+	LinuxOS   = "linux"
+	Amd64Arch = "amd64"
+	FmtOSArch = "%s/%s" // Stringified platform.
 )
 
+var ValidPlatforms = []string{
```

Comment:
can this be private?

Reply:
Yes. I had to make it public when I moved it into the `exec` package, but I can just hard code the value(s) in the unit tests.

Reply:
No worries! It can remain public then 👍